python/ray/air/_internal/usage.py

```diff
@@ -226,6 +227,31 @@ def _get_tag_for_remote_path(remote_path: str) -> str:
     return tag
 
 
+def tag_storage_type(storage: "StorageContext"):
+    """Records the storage configuration of an experiment.
+
+    The storage configuration is set by `RunConfig(storage_path, storage_filesystem)`.
+
+    The possible storage types (defined by `pyarrow.fs.FileSystem.type_name`) are:
+    - 'local' = pyarrow.fs.LocalFileSystem. This includes NFS usage.
+    - 'mock' = pyarrow.fs._MockFileSystem. This is used for testing.
+    - ('s3', 'gcs', 'abfs', 'hdfs'): Various remote storage schemes
+        with default implementations in pyarrow.
+    - 'custom' = All other storage schemes, which includes ALL cases where a
+        custom `storage_filesystem` is provided.
+    """
+    whitelist = {"local", "mock", "s3", "gcs", "abfs", "hdfs"}
```

Comment:
Is there also `gs` as an alternative for `gcs`?

Reply:
Nope, pyarrow fs type name is always gcs